### PR TITLE
[IZPACK-1659] CSS is displayed with `-console`

### PR DESCRIPTION
**Pull request by hiteshbosamiya**

### CODE REVIEW

File: izpack-installer/src/main/java/com/izforge/izpack/installer/console/AbstractTextConsolePanel.java [hunk 1]
```diff
@@ -194,4 +191,11 @@ protected String removeHTML(String text)
         }
         return result;
     }
+
+    private String remoteHTMLTag(String text, String tag)
```
Comment (reinhapa) [inline izpack-installer/src/main/java/com/izforge/izpack/installer/console/AbstractTextConsolePanel.java:195]: I guess you meant `removeHTMLTag` as a method name here?

File: izpack-installer/src/test/java/com/izforge/izpack/installer/console/AbstractTextConsolePanelTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                "<table width=\"100%\" cellpadding=10 border=0>\n" +
+                "<tr>\n" +
+                "<td align=\"right\" valign=\"top\">\n" +
+                "<h1>Welcome to IzPack</h1>\n" +
+                "</td>\n" +
+                "</tr>\n" +
+                "</table>\n" +
+                "</body>\n" +
+                "</html>\n";
+        String expected =  "\n\nWelcome\n\n\n\n\n\r\r\n\t\nWelcome to IzPack\n\n\n\n\n\n";
```
Comment (reinhapa) [inline izpack-installer/src/test/java/com/izforge/izpack/installer/console/AbstractTextConsolePanelTest.java:73]: @akuhtz would it make sense to remove some superfluous new lines here? 
- no leading
- no tailing
- maximum two between the lines

**Review by reinhapa — CHANGES_REQUESTED** [review]

File: izpack-installer/src/test/java/com/izforge/izpack/installer/console/AbstractTextConsolePanelTest.java (hunk 2 above)
Comment (hiteshbosamiya) [reply izpack-installer/src/test/java/com/izforge/izpack/installer/console/AbstractTextConsolePanelTest.java:73]: I think it makes sense to go with these suggestions.

File: izpack-installer/src/main/java/com/izforge/izpack/installer/console/AbstractTextConsolePanel.java (hunk 1 above)
Comment (hiteshbosamiya) [reply izpack-installer/src/main/java/com/izforge/izpack/installer/console/AbstractTextConsolePanel.java]: Yes, corrected the typo.

**Comment by hiteshbosamiya** [comment]
Corrected method name.

**Review by akuhtz — APPROVED** [review]

File: izpack-installer/src/test/java/com/izforge/izpack/installer/console/AbstractTextConsolePanelTest.java (hunk 2 above)
Comment (hiteshbosamiya) [reply izpack-installer/src/test/java/com/izforge/izpack/installer/console/AbstractTextConsolePanelTest.java]: Taken care of this point, please review.

**Review by reinhapa — APPROVED** [review]

**Pull request merged**